types/emoji-mart/dist-es/utils/shared-props.d.ts

```diff
@@ -69,6 +69,7 @@ export interface PickerProps {
     i18n?: PartialI18n;
     style?: React.CSSProperties;
     title?: string;
+    theme?: string;
```

Comment:
A string literal will provide better type checking and autocomplete here.
```suggestion
    theme?: "auto" | "light" | "dark";
```